### ISSUE TITLE
s-table right border is not visible when last column is hidden

### ISSUE THREAD

**Issue by benjamin-hodgson** [StackExchange/Stacks#142]
**To Reproduce**
This is the HTML from [the example in the docs](https://stackoverflow.design/product/components/tables), except I've added `d-none` to the last `th` and `td` of each row.

```html
<table class="s-table">
    <thead>
        <tr>
            <th scope="col">Name</th>
            <th scope="col">Reputation</th>
            <th scope="col">Joined</th>
            <th scope="col" class="d-none">Last Seen</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <th><a>Aaron Shekey</a></th>
            <td><div class="grid ai-center"><div class="fw-bold mr8" title="reputation score " dir="ltr">263</div><div class="grid ai-center"><img class="s-badge--image" src="/assets/img/BadgeBronze.svg" aria-hidden="true"> <span>7</span></div></div></td>
            <td>Dec 1 ’17 at 20:24</td>
            <td class="d-none">just now</td>
        </tr>
        <tr>
            <th><a>Joshua Hynes</a></th>
            <td><div class="grid ai-center"><div class="fw-bold mr8" title="reputation score " dir="ltr">611</div><div class="grid ai-center"><img class="s-badge--image" src="/assets/img/BadgeBronze.svg" aria-hidden="true"> <span>11</span></div></div></td>
            <td>Feb 12 at 18:47</td>
            <td class="d-none">Aug 10 at 14:57</td>
        </tr>
        <tr>
            <th><a>Paweł Ludwiczak</a></th>
            <td><div class="grid ai-center"><div class="fw-bold mr8" title="reputation score " dir="ltr">25</div><div class="grid ai-center"><img class="s-badge--image" src="/assets/img/BadgeBronze.svg" aria-hidden="true"> <span>8</span></div></div></td>
            <td>Nov 28 ’17 at 20:21</td>
            <td class="d-none">1 hour ago</td>
        </tr>
        <tr>
            <th><a>Piper Lawson</a></th>
            <td><div class="grid ai-center"><div class="fw-bold mr8" title="reputation score " dir="ltr">25</div><div class="grid ai-center"><img class="s-badge--image" src="/assets/img/BadgeBronze.svg" aria-hidden="true"> <span>3</span></div></div></td>
            <td>Jul 5 at 14:32</td>
            <td class="d-none">Aug 14 at 12:41</td>
        </tr>
    </tbody>
</table>
```

I can't just omit the `td`s because I need to show and hide them from JS.

**Expected behavior**
The right-hand border should still be visible, like this:
![expected](https://user-images.githubusercontent.com/3857587/45631131-3a2a0e80-ba68-11e8-8749-db2b03c8242e.png)

**Actual behaviour**
Right hand border is missing
![actual](https://user-images.githubusercontent.com/3857587/45631158-49a95780-ba68-11e8-9d98-0aafd2849ed5.png)

**Comment by benjamin-hodgson** [comment]
I experimented with removing the `th:last-of-type` and `td:last-of-type` rules and just setting the `border` on the `table` element instead and it seems to work. Is there a reason it wasn't like that in the first place?

**Comment by aaronshekey** [comment]
Fixed in #146
